api/src/main/java/org/openbaton/nfvo/api/admin/RestVimInstances.java

```diff
@@ -100,6 +98,7 @@ public void delete(
    *
    * @return List<Datacenter>: The List of Datacenters available
    */
+  @ApiOperation(value = "Retrieving all Vim Instances", notes = "")
```

Comment:
Yes. Notes are a more detailed description and I did not know what more to add here.